main.go

```diff
@@ -78,10 +78,10 @@ type manifestLine struct {
 type manifest []manifestLine
 
 const (
-	decredRelver = "v1.7.0-rc2"
-	dexcRelver   = "v0.4.0-rc2"
-	ldVersion    = "1.7.0-rc2"
-	prerelease   = "rc2"
+	decredRelver = "v1.7.0-rc3"
+	dexcRelver   = "v0.4.0-rc3"
+	ldVersion    = "1.7.0-rc3"
```

Comment:
When dex decides to go 1.x (when the APIs and comms protocols stabilize), we can jump straight to 1.7.x or whatever it happens to coincide with.

Reply:
noted, thx